lib/model.go

```diff
@@ -124,3 +124,62 @@ type RenameTopicRequest struct {
 type SubscribersTopicRequest struct {
 	Subscribers []string `json:"subscribers"`
 }
+
+type IntegrationCredentials struct {
+	ApiKey           string                 `json:"apiKey,omitempty"`
+	User             string                 `json:"user,omitempty"`
+	SecretKey        string                 `json:"secretKey,omitempty"`
+	Domain           string                 `json:"domain,omitempty"`
+	Password         string                 `json:"password,omitempty"`
+	Host             string                 `json:"host,omitempty"`
+	Port             string                 `json:"port,omitempty"`
+	Secure           bool                   `json:"secure,omitempty"`
+	Region           string                 `json:"region,omitempty"`
+	AccountSid       string                 `json:"accountSid,omitempty"`
+	MessageProfileId string                 `json:"messageProfileId,omitempty"`
```

Comment:
Kindly update `AccountSid` to `AccountSID`, also `MessageProfileId` to `MessageProfileID`